modules/KalturaSupport/components/chapters/chapters.js

```diff
@@ -481,7 +481,9 @@
 				var customData = item.partnerData ? JSON.parse(item.partnerData) : {};
 				var title = item.title || customData.title;
 				var description = item.description || customData.desc;
-				var thumbnailUrl = item.thumbnailUrl || customData.thumbUrl || _this.getThumbUrl(item);
+				var thumbnailUrl = mw.getConfig('loadThumbnailWithKs')
```

Comment:
this is incorrect! you are now basically making the item.thumbnailUrl and customData.thumbUrl obsolete if the flag is set.
You need to take the respective url and add to it a ks if flag is set and not use the `getThumbUrl` global url settings.

Reply:
Done, also no need to add this logic to the chapters plugin, KS is already added.